earth_enterprise/src/fusion/fusionui/AssetBase.cpp

```diff
@@ -63,12 +63,14 @@ AssetBase::AssetBase(QWidget* parent)
   saveas_action_->setIconSet(QIconSet(QPixmap::fromMimeSource("filesaveas.png")));
   build_action_ = new QAction(this);
   //  build_action_->setIconSet(QIconSet(QPixmap::fromMimeSource("notes.png")));
+  savebuild_action_ = new QAction(this);
   close_action_ = new QAction(this);
   close_action_->setIconSet(QIconSet(QPixmap::fromMimeSource("fileclose.png")));
   hidden_action_ = new QAction(this);
   hidden_action_->setToggleAction(true);
   notes_action_ = new QAction(this);
   notes_action_->setIconSet(QIconSet(QPixmap::fromMimeSource("notes.png")));
+  savebuild_action_ = new QAction(this);
```

Comment:
This is instantiated twice. I'm assuming it's because of a merge conflict. Please remove the redundant instantiation